.github/workflows/ci.yml

```diff
@@ -36,7 +36,6 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - "3.9"
```

Comment:
Please drop support for 3.9 in a separate PR

Reply:
OK I will do that, but until that's merged this PR will fail pre-commit (on github not locally), due to the 3.10 syntax used in flux_led and copied to ProtocolFairy.  [See this build](https://github.com/Bluetooth-Devices/led-ble/actions/runs/12358651496/job/34489669739)

Reply:
Add `from __future__ import annotations` to the top of the file and it will work in 3.9